src/lib/components/EnvironmentLabel/EnvironmentLabel.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Overridable from 'react-overridable';
+import { Icon, Label } from 'semantic-ui-react';
+
+const EnvironmentLabelLayout = ({
+  text,
+  icon,
+  color,
+  pointing,
+  ...uiProps
+}) => {
+  return (
+    <Label
+      color={color}
+      pointing={pointing}
+      size="small"
+      className="environment-label"
+      {...uiProps}
+    >
+      <Icon name={icon} />
+      {text}
+    </Label>
+  );
+};
+
+EnvironmentLabelLayout.propTypes = {
+  text: PropTypes.string.isRequired,
+  icon: PropTypes.string.isRequired,
+  color: PropTypes.string.isRequired,
+  pointing: PropTypes.string.isRequired,
+};
+
+const EnvironmentLabelComponent = ({ pointing, ...uiProps }) => {
+  const environment = process.env.REACT_APP_ENV_NAME;
+  if (!environment || environment === 'development') {
```

Comment:
I would propose to put by default our envs, so we don't have to override :)
or maybe you can add to the `invenioConfig.APP` a new obj/list with all the possible envs, e.g.:
```
invenioConfig.APP.environments: [{
  name: 'dev',
  text: 'Dev environment',
  color: 'blue'
 icon: ...
}]
```
and then here:
```
env = invenioConfig.APP.environments.filter(env => env.name === process.env.REACT_APP_ENV_NAME);
return (
      <EnvironmentLabelLayout
        icon="code"
        text="Development environment"
        color="blue"
        pointing={pointing}
        {...uiProps}
      />
    );
...

```

in CDS-ILS, we will have only to add the config.
WDYT?

I will add now the var `REACT_APP_ENV_NAME` to the deployment

Reply:
What `REACT_APP_ENV_NAME` name did you use in the end? So that we align